eng/pipelines/common/platform-matrix.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       targetRid: linux-musl-arm
       platform: Linux_musl_arm
       container:
-        image: ubuntu-18.04-cross-arm-alpine-20200818211451-14441ae
+        image: ubuntu-18.04-cross-arm-alpine-20210409142425-044d5b9
```

Comment:
Please keep this one unchanged for now , it would break CI test runs for arm alpine. This new image can target Alpine >= 3.13 now. I have a PR to update it properly.